pop-up-details-desktop.js

```diff
@@ -0,0 +1,409 @@
+const bigContainer = document.createElement('div');
+bigContainer.setAttribute('class', 'projectPopUp');
+bigContainer.style.width = '100%';
+bigContainer.style.height = '1851px';
+
+const insideProject = document.createElement('div');
+insideProject.setAttribute('class', 'inside');
+insideProject.style.width = '100%';
+insideProject.style.height = '1711px';
+
+document.body.appendChild(bigContainer);
+bigContainer.append(insideProject);
+
+const myFirstProjectBtn = document.querySelector('.my-first-project-button');
+const mySecondProjectBtn = document.querySelector('.my-second-project-button');
+
+const rectBar = document.createElement('span');
+const rectBar2 = document.createElement('span');
+
+const projectName = document.createElement('h2');
+projectName.setAttribute('class', 'projectTitle');
+projectName.innerHTML = 'Project name goes here';
+
+const progLang = document.createElement('ul');
```

Comment:
Kindly consider moving the close button on the mobile screen to the top right corner of the screen in order to have the correct alignment.

<img width="216" alt="vaaaa" src="https://github.com/norbiboy1/Portfolio/assets/84930355/71e7269c-77e2-4e0d-bcd2-48fa808d1223">